planetiler-core/src/main/java/com/onthegomap/planetiler/reader/GeoPackageReader.java

```diff
@@ -74,8 +74,14 @@ public void readFeatures(Consumer<SimpleFeature> next) throws Exception {
     for (var featureName : geoPackage.getFeatureTables()) {
       FeatureDao features = geoPackage.getFeatureDao(featureName);
 
+      // If left unset (e.g. in NaturalEarth's data), assume latlon
+      long srsId = features.getSrsId();
+      if (srsId == 0) {
+        srsId = 4326;
+      }
+
       MathTransform transform = CRS.findMathTransform(
-        CRS.decode("EPSG:" + features.getSrsId()),
+        CRS.decode("EPSG:" + srsId),
```

Comment:
From [the spec](https://www.geopackage.org/spec131/#r11):

> The record with an srs_id of 0 SHALL be used for undefined geographic coordinate reference systems.

So adding an explicit projection would be required for correctness, but if not given, maybe we can fallback to 4326 rather than throwing?

Reply:
Updated to match shapefile reader behavior (will throw rather than assuming a default projection, can be overridden unconditionally)